tv-show/src/components/features/shows/ShowContainer/ShowContainer.tsx

```diff
@@ -56,26 +38,39 @@ interface IShowContainerProps {
 }
 
 export default function ShowContainer({showData}: IShowContainerProps) {
+  const user = useUser();
+  const remoteReviews = useReviews(showData.id);
```

Comment:
You don't really need to keep internal state anymore. SWR cache is your state now. This just provides an opportunity for the states to go out of sync.